app/controllers/api/v1/credit_notes_controller.rb

```diff
@@ -93,6 +93,9 @@ def void
       end
 
       def index
+        billing_entities = current_organization.billing_entities.where(code: params[:billing_entity_codes]) if params[:billing_entity_codes].present?
+        return not_found_error(resource: "billing_entity") if params[:billing_entity_codes].present? && billing_entities.count != params[:billing_entity_codes].count
```

Comment:
Do we want to fail or return results as long as there is at least one match?

Reply:
hah, initially I had exactly this logic, but then I decided that since for not existing we return `not_found`, it will be better if we will return `not_found` in this case as well, so users will know that they sent something wrong